src/main/java/org/owasp/validator/html/Policy.java

```diff
@@ -735,6 +766,25 @@ public Attribute getGlobalAttributeByName(String name) {
     }
 
     /**
+     * A method for returning one of the dynamic <global-attribute> entries by
+     * name.
+     *
+     * @param name The name of the dynamic global-attribute we want to look up.
+     * @return An Attribute associated with the global-attribute lookup name specified,
+     * or null if not found.
+     */
+    public Attribute getDynamicAttributeByName(String name) {
+        Attribute dynamicAttribute = null;
+        for (String d : dynamicAttributes.keySet()) {
+            if (name.startsWith(d)) {
```

Comment:
Yeah, sorry what do you mean by retarget? It should merge clean to master and it could be in any branch. It could be used for any attributes that are varied. Another one that came up lately was aria-* where there are a lot of attributes to support a11y aria tags, and it's a pain to try to define them all.